pynih/source/python/conv/d_to_python.d

```diff
@@ -77,9 +77,10 @@ PyObject* toPython(T)(T value) if(is(Unqual!T == Date)) {
 }
 
 
-PyObject* toPython(T)(T value) if(is(T == string)) {
+PyObject* toPython(T)(T value) if(isSomeString!T) {
     import python.raw: pyUnicodeFromStringAndSize;
-    return pyUnicodeFromStringAndSize(value.ptr, value.length);
+    import std.conv: to;
+    return pyUnicodeFromStringAndSize(value.to!string.ptr, value.length);
```

Comment:
`value.length` may not match `value.to!string.length`

Reply:
Sigh. The page just refreshed as I merged and I saw your comment afterwards.